flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -748,7 +748,7 @@ function retry_times_with_backoff_and_cleanup() {
     local command="$3"
     local cleanup_command="$4"
 
-    for (( i = 0; i < ${retriesNumber}; i++ ))
+    for i in $(seq 1 ${retriesNumber})
```

Comment:
Why is this better than the previous for loop version?

Reply:
This block of loop will only executes 1 time in this scenario. I found that there is also a for loop  in start_confluent_schema_registry function:
```
for i in {1..30}; do
    if get_and_verify_schema_subjects_exist; then
        echo "Schema registry is up."
        return 0
    fi
    echo "Waiting for schema registry..."
    sleep 1
  done
```
The variable i  is set to 30 after this function, which cause the loop end after only 1 execute.

Reply:
I tried to reproduce the problem via

```
for i in {1..30}; do
  echo $i
done

for (( i = 0; i < 10; i++ )) do
  echo $i
done
```

When I execute this, then 1..30 and 0..9 will be printed. Hence, I'm not sure I really understand the problem you are describing. Could you provide me an example script to reproduce the problem with the old for loop?

Reply:
I think current scenario equals to the following script:
```
for (( i = 0; i < 10; i++ )) do
  for i in {1..30}; do
     echo $i
  done
  echo $i
done
```

Reply:
Ah this makes sense. Thanks a lot for your explanation.